### PR TITLE
Merge the `dev` branch into the `master` branch

### PR DESCRIPTION
**Pull request by DilumAluthge**
Includes the following pull requests:
1. Fix the docs build (#483)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -62,19 +62,23 @@ jobs:
...
-           build_docs = (base_ref == "master") && (head_ref == "dev") && (base_repository == head_repository)
+           event_name = "${{ github.event_name }}"
+           if event_name == "pull_request"
+               base_ref = "${{ github.base_ref }}"
+               head_ref = "${{ github.head_ref }}"
+               base_repository = "${{ github.repository }}"
+               head_repository = "${{ github.event.pull_request.head.repo.full_name }}"
+               build_docs = (base_ref == "master") && (head_ref == "dev") && (base_repository == head_repository)
+           else 
+               build_docs = true
```
Comment (ablaom) [inline .github/workflows/ci.yml:73]: To me this looks like docs get built when for any non-`pull_request` event, including `push` events (commits pushed to any branch). I don't want this. Am I missing something?

**Comment by ablaom** [comment]
CI still not seeing Distributions 0.24.5

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ablaom) [inline .github/workflows/ci.yml:73]:
```suggestion
               build_docs = false
```

Comment (DilumAluthge) [reply .github/workflows/ci.yml:73]: I would do https://github.com/alan-turing-institute/MLJBase.jl/pull/485 instead. You still want to build docs on pushes to master, right? And presumably for tags also.

Comment (ablaom) [reply .github/workflows/ci.yml:73]: Yes and yes.

**Pull request closed without merging**